### ISSUE TITLE
Send to Cgeo registration window not displaying in Firefox

### ISSUE THREAD

**Issue by vicarvernon** [cgeo/send2cgeo#57]
> http://www.cgeo.org/send2cgeo
on Firefox 65.0.1 (64bit)
registration window not displaying
Text message "Your Connection is not secure"
![image](https://user-images.githubusercontent.com/36171602/52707199-11f2fd00-2f7f-11e9-804e-27c09516a22a.png)
(tried to provide a screenshot - hope you can see)

webpage displays fine under Edge

Windows 10 Home
version 1809
OS Build 17763.292

**Comment by SammysHP** [comment]
It is not possible to attach files via email.

**Comment by vicarvernon** [comment]
Same result as in the window

**Comment by mucek4** [comment]
I have just tried in Firefox 65.0.1 (64bit) and it work without a problem. Looks like some sofr of problem on your side. Until you provide more data (learn more, advanced, ...) we cannot help you.

**Comment by vicarvernon** [comment]
As a user of the website I can do without rudeness ("learn more" !).
Without guidance, how can learning successfully take place.

I have a brand new PC. I downloaded the lastest version of Firefox browser.
The Cgeo website should just work. The error message indicates it's a
server side issue, not a client side issue in the way the site is
configured for it's certificate that prevents it from showing correctly.

That's my understanding as a non technical user. I recognize Cgeo is run by
volunteers and am thankful for it as the means of 10 years of geocaching.

I have posted on here for help and to alert to an issue.

What further information is required?

**Comment by kumy** [comment]
@vicarvernon I just check and it's also working for me. may you please open the direct url https://send2.cgeo.org/api/ , does it works? If not, click "Advanced" -> "view certificate", can you post screenshot using github website (not mail). Thanks

**Comment by vicarvernon** [comment]
seems to be working fine now
successfully registered device

Thank you all.

**Comment by mucek4** [comment]
Rudeness? On the screenshot you provided you have a link and a button. One is called "learn more" and other one is called "advanced". To help you resolve a problem I would need screenshots of pages under those two links. Maybe I expressed a bit too technical but I provided a guidance how to resolve a problem.
If it started working then probably your FF updated it's certificates. We didn't changed server side at all.
